pkg/util/fedinformer/keys/keys.go

```diff
@@ -74,7 +74,18 @@ func ClusterWideKeyFunc(obj interface{}) (ClusterWideKey, error) {
 		return key, fmt.Errorf("object has no meta: %v", err)
 	}
 
+	// When using a typed client, decoding to a versioned struct (not an internal API type), the apiVersion/kind
```

Comment:
Is APIVersion/Kind information of `runtime.Object` filled in when we using dynamic client?